09_palindromes/palindromes.js

```diff
@@ -1,4 +1,20 @@
-const palindromes = function () {
+const palindromes = function (str) {
+
+    let cleanStr = str.replace(/\W/g, '');
+    let upperStr = cleanStr.toUpperCase()
+    let reverseStr = ""
+    for (let i = upperStr.length - 1; i >= 0; i--) {
+        reverseStr += upperStr[i]
+
+    }
+
+    if (upperStr != reverseStr) {
+        return false
+    }
+    else {
+        return true
+    }
+
```

Comment:
Return the if condition itself!
`return upperStr != reverseStr`

Reply:
done